src/app/loadout-drawer/LoadoutDrawer2.tsx

```diff
@@ -315,7 +330,8 @@ async function pickLoadoutItem(
           item.classType === loadoutClassType ||
           item.classType === DestinyClass.Unknown) &&
         itemCanBeInLoadout(item) &&
-        !loadoutHasItem(item),
+        !loadoutHasItem(item) &&
+        (item.notransfer ? item.owner === store.id : true),
```

Comment:
```suggestion
        !loadoutHasItem(item) &&
        (!item.notransfer || item.notransfer && item.owner === store.id),
```

Reply:
Ah much better! I think it can be simplified to `(!item.notransfer || item.owner === store.id)` since the or short-circuits if the first is true. Thanks!